### PR TITLE
CNCF Kubernetes Blog Cross-Post

### PR DESCRIPTION
**Pull request by CazJT**

### CODE REVIEW

File: blog/extending-kubernetes-frontiers-components/index.mdx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+title: 'How WebAssembly components extend the frontiers of Kubernetes to multi-cloud, edge, and beyond'
+image: '/img/next-wave-header.jpg'
+date: 2024-07-08T09:00:00+05:00
+author: 'Taylor Thomas'
+author_profile: 'https://twitter.com/oftaylor'
+description: 'How WebAssembly components are unlocking new ways to extend the value of existing Kubernetes investments'
+draft: false
+---
+
+WebAssembly (Wasm) components are here and [already unlocking new computing patterns](previous-CNCF-piece-link-TK). Meanwhile, CNCF's [**wasmCloud**](https://wasmcloud.com/) offers Wasm-native orchestration for distributed components&mdash;in essence, a Kubernetes for WebAssembly.
```
Comment (lachieh) [inline blog/extending-kubernetes-frontiers-components/index.mdx]:
```suggestion
WebAssembly (Wasm) components are here and [already unlocking new computing patterns](#previous-CNCF-piece-link-TK). Meanwhile, CNCF's [**wasmCloud**](https://wasmcloud.com/) offers Wasm-native orchestration for distributed components&mdash;in essence, a Kubernetes for WebAssembly.
```

just changing for build

File: blog/extending-kubernetes-frontiers-components/index.mdx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+title: 'How WebAssembly components extend the frontiers of Kubernetes to multi-cloud, edge, and beyond'
+image: '/img/next-wave-header.jpg'
+date: 2024-07-08T09:00:00+05:00
+author: 'Taylor Thomas'
+author_profile: 'https://twitter.com/oftaylor'
+description: 'How WebAssembly components are unlocking new ways to extend the value of existing Kubernetes investments'
+draft: false
+---
+
+WebAssembly (Wasm) components are here and [already unlocking new computing patterns](#previous-CNCF-piece-link-TK). Meanwhile, CNCF's [**wasmCloud**](https://wasmcloud.com/) offers Wasm-native orchestration for distributed components&mdash;in essence, a Kubernetes for WebAssembly.
```
Comment (lachieh) [inline blog/extending-kubernetes-frontiers-components/index.mdx:12]: This link will need to be corrected before merge

**Review by lachieh — CHANGES_REQUESTED** [review]
Looks good, requesting changes to prevent merge with missing link

**Comment by CazJT** [comment]
@lachieh changed the link to one of Bailey's talks as it is excellent and also contains a longer blog on the page. Check everything is in order. I adjusted in codespace and updated the branch. This hasn't been working amazingly and so there may be issues.

File: blog/extending-kubernetes-frontiers-components/index.mdx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
```
Comment (lachieh) [inline blog/extending-kubernetes-frontiers-components/index.mdx:13]:
```suggestion

{/* truncate */}

```

**Review by lachieh — APPROVED** [review]

**Pull request merged**